apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.tsx

```diff
@@ -401,93 +415,235 @@ onDocumentReady(() => {
-            $(this).parents('form').find('.reopen-closed-field').toggle(this.value !== '+1' && this.value !== '-1' && this.value !== 'expire');
-            "
-        >
-          ${row.time_remaining_sec !== null
-            ? row.has_open_instance
-              ? html`
-                  <option value="+1">Add to instances with time limit</option>
-                  <option value="-1">Subtract from instances with time limit</option>
-                `
-              : html`
-                  <option value="+1">Add</option>
-                  <option value="-1">Subtract</option>
-                `
-            : ''}
-          <option value="set_total">Set total time limit to</option>
-          <option value="set_rem">Set remaining time to</option>
-          ${row.open || row.time_remaining_sec !== null
-            ? html`<option value="unlimited">Remove time limit</option>`
-            : ''}
-          ${row.open !== false && (row.time_remaining_sec === null || row.time_remaining_sec > 0)
-            ? html`<option value="expire">Expire time limit</option>`
-            : ''}
-        </select>
-        <p class="form-inline">
-          <input
-            class="form-control time-limit-field"
-            type="number"
-            name="time_add"
-            aria-label="Time value"
-            style="width: 5em"
-            value="5"
-          />
-          <select class="custom-select time-limit-field" name="time_ref" aria-label="Time unit">
-            <option value="minutes">minutes</option>
-            ${row.time_remaining_sec !== null
-              ? html`<option value="percent">% total limit</option>`
-              : ''}
-          </select>
-        </p>
-        ${row.has_closed_instance
-          ? html`
-              <div class="form-check mb-2 reopen-closed-field">
+  function TimeLimitExplanation({ action }: { action: TimeLimitAction }) {
+    let explanation = '';
+    if (action === 'set_total') {
+      explanation =
+        'Updating the total time limit will set the given amount of time for the assessment based on when the assessment was started.';
+    } else if (action === 'set_rem') {
+      explanation =
+        'Updating the time remaining will set the given amount of time for the assessment based on the current time.';
+    } else if (action === 'set_exact') {
+      explanation = 'This will set the exact closing time for the assessment.';
+    } else if (action === 'add') {
+      explanation = 'This will add the given amount of time to the remaining time limit.';
+    } else if (action === 'subtract') {
+      explanation = 'This will subtract the given amount of time from the remaining time limit.';
+    } else if (action === 'remove') {
+      explanation = 'This will remove the time limit and the assessment will remain open.';
+    } else if (action === 'expire') {
+      explanation = 'This will expire the time limit and the assessment will be closed.';
```

Comment:
I don't think this is true; removing access does not automatically close the assessment. Remember that whether an assessment is open is orthogonal to whether it has an access time limit.

Reply:
Updated to 
`'This will expire the time limit and students will be unable to submit any further answers.'` 
Is this more appropriate? Tried to summarize from [the docs](https://prairielearn.readthedocs.io/en/latest/accessControl/#time-limit-adjustments-for-open-assessments). If there is better verbiage for this, let me know.